platform/security/encryption/platform-security-encryption-commands/src/test/java/ddf/security/command/EncryptCommandTest.java

```diff
@@ -16,34 +16,37 @@
 import static org.junit.Assert.assertNull;
 import static org.junit.Assert.fail;
 
+import ddf.security.SecurityConstants;
 import ddf.security.encryption.impl.EncryptionServiceImpl;
-import java.io.File;
 import java.lang.reflect.Field;
-import java.nio.file.Files;
-import org.apache.commons.io.FileUtils;
 import org.junit.After;
 import org.junit.Before;
 import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 public class EncryptCommandTest {
   private static final Logger LOGGER = LoggerFactory.getLogger(EncryptCommandTest.class);
 
-  private static File ddfHome;
+  private static final TemporaryFolder TEMPORARY_FOLDER = new TemporaryFolder();
```

Comment:
❓ Why are you managing your own temp folder creation & deletion?
https://junit.org/junit4/javadoc/4.12/org/junit/rules/TemporaryFolder.html